test/tmt/main.fmf

```diff
@@ -0,0 +1,33 @@
+# Only common dependencies that are NOT required to run netavark-tests.sh are
+# specified here. Everything else is in netavark-tests.sh.
+require:
+    - cargo
+    - make
+
+adjust:
+    duration: 10m
+    when: arch == aarch64
```

Comment:
Why only on aarch?

Reply:
IIRC, the default is 5m and that sufficed for x86_64 consistently. Only aarch64 timed out sometimes.